back/app/controllers/web_api/v1/projects_controller.rb

```diff
@@ -20,17 +20,10 @@ def index
     # scope.
 
     @projects = Project.where(id: publications.select(:publication_id))
+      .ordered
       .includes(:project_images, :phases, :areas, admin_publication: [:children])
     @projects = paginate @projects
 
-    @projects = if params[:search].present?
-      @projects.search_by_all(params[:search])
-    else
-      @projects.ordered
-    end
-
-    LogActivityJob.perform_later(current_user, 'searched_projects', current_user, Time.now.to_i, payload: { search_query: params[:search] }) if params[:search].present?
-
```

Comment:
@lilkraftwerk Maybe we could add something like this to the admin_publications_controller? Would be nice to have some data on usage.

Reply:
Will do so now, I'll re-request review so you can take a look!